### ISSUE TITLE
When tasks are created, create associations between themselves and their creators

### ISSUE THREAD

**Issue by jmeuwissen** [calahhansen/Doppelganger-run#38]
Will probably be a series of callbacks to identify which task was just created, or perhaps we can directly set the foreign key associated with the "creator" user. From what I can gather via mysql workbench, Sequelize represents its "hasMany" and "belongsTo" associations by adding foreign key columns to the many side of the "one-to-many" relationship. In our case, the foreign keys of our "Task" table/model are labeled as "assigneeId" and "UserId" for the "assignee" and "creator" associations, respectively. 

tldr: when we create a task in our route, we could directly set UserId foreign key instead of using Sequelize's associate methods
